editor/inspector_dock.cpp

```diff
@@ -788,12 +788,12 @@ InspectorDock::InspectorDock(EditorData &p_editor_data) {
 	unique_resources_list_tree->set_auto_translate_mode(AUTO_TRANSLATE_MODE_DISABLED);
 	unique_resources_list_tree->set_hide_root(true);
 	unique_resources_list_tree->set_columns(1);
-	unique_resources_list_tree->set_column_title(0, TTR("Property"));
+	unique_resources_list_tree->set_column_title(0, TTRC("Property"));
```

Comment:
This line seems to be a remnant from a previous refactoring and can be deletely directly. This `Tree` only has one column and the column title is never shown.